bundles/org.openhab.binding.daikin/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -98,6 +100,20 @@
 		</state>
 	</channel-type>
 
+	<channel-type id="acunit-homekitmode">
+		<item-type>String</item-type>
+		<label>Homekit Mode</label>
+		<description>Current Homekit mode of the AC unit</description>
+		<state>
+			<options>
+				<option value="on">Auto</option>
```

Comment:
```suggestion
				<option value="auto">Auto</option>
```

Reply:
Will fix shortly - at one point I read the docs and thought there was no auto, and it had to be 'on'. Now the docs say 'By default the integration will provide "off,heat,cool,on,heatcool,auto,eco".'

Reply:
What is the difference between heatcool and auto?

Reply:
I don't think there's any difference between heatcool and auto - just google being google ;)

Reply:
My guess is:
on->fan on
heatcool -> auto heating and cooling without fan
auto -> auto heating and cooling with fan
eco -> auto fan (not sure if this is correct)